CMakePresets.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
nice ! - Doesnt this require a comment header?

Reply:
Ah, actually I added that for testing (that's why it's bare bones). I'll pull it out, completely and add it back in in a follow up.

No there are no comments in JSON so we have to exclude it from the check (or it seems to already be).